test/smart_collection_test.rb

```diff
@@ -7,4 +7,29 @@ class SmartCollectionTest < Test::Unit::TestCase
     smart_collection = ShopifyAPI::SmartCollection.create(:title => "Macbooks", :rules => rules)
     assert_equal 1063001432, smart_collection.id
   end
+
+  test "Smart Collection get products gets all products in a smart collection" do
+    fake "smart_collections/1063001432", method: :get, status: 200, body: load_fixture('smart_collection')
+    smart_collection = ShopifyAPI::SmartCollection.find(1063001432)
```

Comment:
Could you do `ShopifyAPI::SmartCollection.` instead of faking the endpoint data?